nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,78 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.listen(3000, () => console.log(`Calculator:listening on port 3000`));
+//Added 4 routes to index.js:
+//GET /calculator/add?first=<number here>&second=<number here>.
+//In response send sum (first + second).
+app.get("/calculator/add", async (request, response) => {
+  const first = request.query.first;
+  const second = request.query.second;
+  const sum = arrayContentAddition(first) + arrayContentAddition(second);
+  response.send(sum.toString());
+});
+//GET /calculator/subtract?first=<number here>&second=<number here>.
+//In response send subtract(first + second).
+app.get("/calculator/subtract", async (request, response) => {
+  const first = request.query.first;
+  const second = request.query.second;
+  const subtract =
+    arrayContentSubtraction(first) - arrayContentSubtraction(second);
+  response.send(subtract.toString());
+});
+
+//GET /calculator/multiply/<first number here>/<second number here>.
+//In response send multiplication (first * second).
+app.get("/calculator/multiply", async (request, response) => {
+  const first = request.query.first;
+  const second = request.query.second;
+  const multiply =
+    arrayContentMultiplier(first) * arrayContentMultiplier(second);
+  response.send(multiply.toString());
+});
+//GET /calculator/divide/<first number here>/<second number here>.
+//In response send divide (first * second).
+app.get("/calculator/divide", async (request, response) => {
+  const first = request.query.first;
+  const second = request.query.second;
+  const divide = arrayContentDivision(first) / arrayContentDivision(second);
+  response.send(divide.toString());
+});
+function arrayContentAddition(arrayVar) {
```

Comment:
Nice idea to pull out some logic into smaller functions! 
Two nitpicks: Try to put blank lines between blocks ({}), it makes it easier to see where a function starts and end. And the name doesn't really explain what it does, it does tell me it's something about arrays but I don't think that's the most important part. 

So I'd find a name which relates more to the domain which is numbers and addition. 

Finally, I think some of this code could be easier written as this:
```js
const values = Array.isArray(param) ? param : [param];
const numbers = values.map(parseInt);
return numbers.reduce((sum, number) => sum + number, 0);
```